src/relay/relay.py

```diff
@@ -522,13 +534,27 @@ def get_trustline_events(
         type: str = None,
         from_block: int = 0,
     ):
-        proxy = self.get_event_selector_for_currency_network(network_address)
+        if type is None:
+            event_types = None
+        else:
+            event_types = [type]
+        proxy: Union[ethindex_db.EthindexDB, CurrencyNetworkProxy]
+        if self.use_eth_index:
+            proxy = ethindex_db.EthindexDB(
+                ethindex_db.connect(""),
+                address=network_address,
+                standard_event_types=currency_network_events.trustline_event_types,
+                event_builders=currency_network_events.event_builders,
+                from_to_types=currency_network_events.from_to_types,
+            )
+        else:
+            proxy = self.currency_network_proxies[network_address]
```

Comment:
This is a duplication of `get_event_selector_for_currency_network` except for `standard_event_types`, I don't fully get the purpose of `standard_event_types` right now but could this be avoided?

Reply:
This will be cleaned up anyways when we remove the proxies, so I think it is fine for now.